### PR TITLE
ItR-12: add lang switcher to the header.

### PR DESCRIPTION
**Pull request by MilaKolasava**
image
image
image

### CODE REVIEW

File: src/components/Toggle/Toggle.tsx [hunk 1]
```diff
@@ -30,7 +33,9 @@ function Toggle(props: TogglesProps) {
         }`}
         data-testid="toggle-switch"
       >
-        {getColorTheme(props.isDarkThemeOff).toUpperCase()}
```
Comment (glebpigulevsky) [inline src/components/Toggle/Toggle.tsx:33]: Why did you delete your function? Keep using it please

**Review by glebpigulevsky — CHANGES_REQUESTED** [review]
Could you add indication to the button. Test to the buttonlang component.

File: src/components/Toggle/Toggle.tsx (hunk 1 above)
Comment (MilaKolasava) [reply src/components/Toggle/Toggle.tsx:33]: I can't use this fanction in this part.

Comment (MilaKolasava) [reply src/components/Toggle/Toggle.tsx:33]: I made a mistake I can do it

File: src/components/ButtonLang/ButtonLang.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  setLocale: React.Dispatch<React.SetStateAction<string>>;
+  locale: string;
+}
+
+function ButtonLang(props: ButtonsLangProps) {
+  return (
+    <>
+      <div
+        className={`button-change button-change-en ${
+          props.locale === LOCALES.ENGLISH ? "active" : ""
```
Comment (glebpigulevsky) [inline src/components/ButtonLang/ButtonLang.tsx]: please, create an function to avoid duplicating of code

Comment (MilaKolasava) [reply src/components/ButtonLang/ButtonLang.tsx]: done

**Review by glebpigulevsky — APPROVED** [review]
гуд жоп
